buffer/src/test/java/io/netty/buffer/SlicedByteBufTest.java

```diff
@@ -138,16 +138,12 @@ public void testForEachByte2() {
         // Ignore for SlicedByteBuf
     }
 
-    @Test(expected = UnsupportedOperationException.class)
     @Override
     public void testDuplicateCapacityChange() {
-        super.testDuplicateCapacityChange();
     }
 
-    @Test(expected = UnsupportedOperationException.class)
     @Override
     public void testRetainedDuplicateCapacityChange() {
-        super.testRetainedDuplicateCapacityChange();
```

Comment:
@Scottmitch Please add a comment in the empty body why this is done so we remember later as well (for all of these methods).